.github/workflows/android.yml

```diff
@@ -20,8 +20,8 @@ jobs:
       - name: Uninstall NDK
         run: sudo ${ANDROID_HOME}/tools/bin/sdkmanager --uninstall `ndk-bundle`
 
-      - name: Build with Gradle
-        run: ./gradlew build
```

Comment:
This failed, it was unable to build a production variant due to a dependency issue. I changed this out for the internalDebug variant until we get closer to completion.

Reply:
👍